build.gradle

```diff
@@ -38,35 +39,36 @@ dependencies {
 }
 
 processResources {
-	inputs.property 'version', project.version
-
-	from(sourceSets.main.resources.srcDirs) {
-		include 'fabric.mod.json'
-		expand 'version': project.version
-	}
-
-	from(sourceSets.main.resources.srcDirs) {
-		exclude 'fabric.mod.json'
+	filesMatching("fabric.mod.json") {
+		expand "version": project.version,
+			"loader_version": project.loader_version,
+			"fabric_kotlin_version": project.fabric_kotlin_version,
+			"mc_version": project.minecraft_version,
+			"mod_id": project.mod_id,
+			"mod_name": project.mod_name,
+			"github_url": project.github_url,
+			"icon_path": project.icon_path
 	}
 }
 
 // ensure that the encoding is set to UTF-8, no matter what the system default is
 // this fixes some edge cases with special characters not displaying correctly
 // see http://yodaconditions.net/blog/fix-for-java-file-encoding-problems-with-gradle.html
-tasks.withType(JavaCompile) {
-	options.encoding = 'UTF-8'
+compileJava {
+	options.encoding = "UTF-8"
```

Comment:
Please don't convert single quotes to double quotes

Reply:
Done in 0b5a6b8